src/common/utility.h

```diff
@@ -347,7 +347,7 @@ OCSYNC_EXPORT Q_DECLARE_LOGGING_CATEGORY(lcUtility)
     }
 
     template <class E>
-    QString enumToString(E value)
+    OCSYNC_EXPORT QString enumToString(E value)
```

Comment:
One compile issue is that you added the export macro to an inline function

Reply:
it was the only issue, and actually the compiler error was quite exact 😅

Reply:
```suggestion
    QString enumToString(E value)
```